ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-stable.yaml

```diff
@@ -117,6 +117,64 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-ovn-ipsec-to-multiarch-f28
```

Comment:
We are placing jobs by job name alphabet order. so need to adjust the newly jobs' place.

Reply:
ack :ok_hand: